genie-agent/build.gradle

```diff
@@ -0,0 +1,32 @@
+dependencies {
+    /*******************************
+     * Compile Dependencies
+     *******************************/
+
+    compile project(":genie-common")
+    compile project(":genie-proto")
+
+    // Logging
+    compile("org.slf4j:slf4j-api")
```

Comment:
You really shouldn't need this. Spring boot starter comes with the spring boot logging module but you can explicitly add the boot logging starter if you want.

Reply:
Done (I copied from genie-client)